README.md

```diff
@@ -1 +1,161 @@
+  </ul>
+</details>
+<details>
+  <summary>Server</summary>
+  <ul>
+    <li>Ruby on Rails</li>
+  </ul>
+</details>
+<details>
+<summary>Database</summary>
+  <ul>
+    <li>Postgresql</li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+- We can add, remove, update and display recipe.
+- It has user authentication.
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## :computer: Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+In order to run this project you need:
+```
+ - Laptop/Desktop.
+ - Internet.
+ - Ruby.
+ - Rails.
+ - Vs Code.
+```
+### Setup
+Clone this repository to your desired folder:
+```
+ - mkdir my-folder
+ - cd my-folder
+ - https://github.com/Ridwanullahi-code/RecipeApp.git
+ - cd Recipe-app
+ - bundle install
+ - rails db:create
+ - rails db:migrate
+```
+### Usage
+
+To run the project, execute the following command:
+- rails server
+
+### Run tests
+
+- rspec.
+
+### Check Linter
+
+- rubocop
+- npx stylelint "**/*.{css,scss}"
+
+### Deployment
+
+- Netlify
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Walid Kabboussa**
+
+- GitHub: [@Hisoka37](https://github.com/Hisoka37)
+- Twitter: [@KBwalid9](https://twitter.com/KBwalid9)
+- LinkedIn: [Walid Kabboussa](https://www.linkedin.com/in/walidkb/)
+
+
+👤 **Ajayi Ridwan**
+
+- Github: [Ridwan](https://github.com/Ridwanullahi-code)
+- LinkedIn: [Ridwan](https://www.linkedin.com/in/ajayi-ridwan/)
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## :telescope: Future Features <a name="future-features"></a>
+- **Enhance design.**
+- **Connect to the Database.**
+- **Add API endpoits.**
+- **Create Responsive Interface.**
+- **Add more animations and transitions.**
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## :handshake: Contributing <a name="contributing"></a>
+Contributions, issues, and feature requests are welcome!
+Feel free to check the [issues page](https://github.com/Ridwanullahi-code/RecipeApp/issues).
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## :star:️ Show your support <a name="support"></a>
+If you like this project you can fork it.
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## :pray: Acknowledgments <a name="acknowledgements"></a>
+I would like to thank... <a href="https://www.microverse.org/?gclid=CjwKCAiArY2fBhB9EiwAWqHK6s-2-x4d57Pghz47XT1BgsYuF81ZprM-k-IwzI0_L96nV0SQ93A8ExoCVnQQAvD_BwE" title="planet icons">Microvers for giving me this opportunity to build this project</a>
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./MIT.md) licensed.
```

Comment:
- [ ] your MIT link is broken, kindly fix that

Reply:
Fixed 👌🏻